aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/actions/SRPCognitoActions.kt

```diff
@@ -30,16 +42,16 @@ object SRPCognitoActions : SRPActions {
         Action<AuthEnvironment>("InitSRPAuth") { id, dispatcher ->
             logger?.verbose("$id Starting execution")
             val evt = try {
-                srpHelper = SRPHelper(event.username, event.password)
+                srpHelper = SRPHelper(event.password)
 
-                val secretHash = SRPHelper.getSecretHash(
+                val secretHash = AuthHelper.getSecretHash(
                     event.username,
                     configuration.userPool?.appClient,
                     configuration.userPool?.appClientSecret
                 )
 
-                var authParams = mapOf("USERNAME" to event.username, "SRP_A" to srpHelper.getPublicA())
-                secretHash?.also { authParams = authParams.plus("SECRET_HASH" to secretHash) }
+                var authParams = mapOf(KEY_USERNAME to event.username, KEY_SRP_A to srpHelper.getPublicA())
+                secretHash?.also { authParams = authParams.plus(KEY_SECRET_HASH to secretHash) }
```

Comment:
should be using 'it' instead of secretHash in the also block. Prefer let instead as well.